### DISCUSSION TITLE
Enclosure Size

### DISCUSSION THREAD

**Discussion by neotil1** [srozum/film_camera_tester#6] [Q&A]
Hi! My "blue metal project box" just arrived and it's waaay too small! I thought they would all be the same size, but I just wasn't paying attention when ordering.

What are the dimensions of the box(es) you used for your enclosure? I wasn't able to find any details about that in the Wiki, unless I missed something.

Thanks!

**Comment by srozum** [comment]
On wiki page https://github.com/srozum/film_camera_tester/wiki/Enclosure you can see photos of build Testers and it will give you an idea what versions of enclosures being used.

**Comment by polygonfuture** [comment]
Hey @neotil1 If you open up the faceplate stl files (vertical and horizontal) it will give you an idea of the height / width dimensions. 

I may be putting together models of all the pcb assemblies + components in Fusion 360 in order to make a custom enclosure for the tester.    If so, I'll see if that's something Serhiy is interested in adding to the repo via pull request.

**Comment by 1N4003** [comment]
For the horizontal version the dimensions are: 115x310x285mm.
